src/Blockcore.Indexer/appsettings.json

```diff
@@ -18,10 +18,10 @@
       }
    },
    "Indexer": {
-      "ConnectionString": "mongodb://{Symbol}-mongo",
+      "ConnectionString": "mongodb://localhost",
       "DatabaseNameSubfix": false,
 
-      "RpcDomain": "{Symbol}-chain",
+      "RpcDomain": "127.0.0.1",
```

Comment:
Don't put these into appsettings.json, use the appsettings.Development.json to override local settings. I think those values are already overwritten in the existing .Development json.